spec/features/sign_in_spec.rb

```diff
@@ -69,14 +70,23 @@
 
       context "Rollover is enabled" do
         let(:user) { build(:user, :new) }
+        let(:transitioned_user) { build(:user, :transitioned, id: user.id) }
+
         let(:user_update_request) do
           stub_request(
             :patch,
             "#{Settings.manage_backend.base_url}/api/v2/users/#{user.id}",
           )
                                       .with(body: /"state":"transitioned"/)
         end
-        let(:user_get_request) { stub_api_v2_request("/users/#{user.id}", user.to_jsonapi) }
+
+        let(:user_get_request) do
+          stub_request(:get, "#{Settings.manage_backend.base_url}/api/v2/users/#{user.id}").to_return(
+            { body: user.to_jsonapi.to_json, headers: { 'Content-Type': "application/vnd.api+json" } },
+            { body: user.to_jsonapi.to_json, headers: { 'Content-Type': "application/vnd.api+json" } },
+            { body: transitioned_user.to_jsonapi.to_json, headers: { 'Content-Type': "application/vnd.api+json" } },
```

Comment:
This spec is transitioning the state of the user, hence why a modified user (i.e. `transitioned_user`) is being returned